lib/rbac/filterer.rb

```diff
@@ -514,6 +517,10 @@ def apply_scope(klass, scope)
       end
     end
 
+    def apply_select(klass, scope, extra_cols)
+      scope.select(klass.arel_table[Arel.star]).select(extra_cols)
```

Comment:
Hmm, possibly.  Since I am still uncertain of the overall affects of the `select` portion of things, I definitely wanted this as a "opt-in" option, so I don't think I will back down on that decision.

That said, I will have to mull this over a bit and see, and you probably are right with the suggestions, I just need to digest them a bit.